docs/sdks/server-sdks/python.md

```diff
@@ -67,56 +72,62 @@ For the assignment event to send the relevant information, we have to expand the
 We have also stored out SDK Key into the environment variable `EPPO_API_KEY` as a common safety practice.
 
 ```python
-import logging, os
+import logging
```

Comment:
Per PEP8, importing multiple libraries on the same line should be avoided https://stackoverflow.com/questions/24271961/is-there-anything-bad-about-having-multiple-imports-on-one-line

Reply:
I’m a savage and refuse to listen to any update to Pep since 2018. Walrus operators are an abomination in the eyes of the BDFL.

Reply:
> I’m a savage and refuse to listen to any update to Pep since 2018. Walrus operators are an abomination in the eyes of the BDFL.

I agree with you on `that := the walrus operator`. Besides `that`, I generally find that the PEP8 guidelines result in more readable code, especially when it comes to stuff like character limits on lines and indentation

Reply:
Definitely